.github/workflows/ci-cd.yml

```
@@ -0,0 +1,125 @@
+      - name: 设置Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+          cache: 'npm'
+      
+      - name: 安装依赖
+        run: npm ci
+      
+      - name: 运行Lint检查
+        run: npm run lint || true
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)